### ISSUE TITLE
Hello, may I ask if you have any pre-training parameters that have not been released? Why is r1 only 0.2 when I use the model to run msrvtt data, is the cross_pytorch_model.bin file not released?

### ISSUE THREAD

**Issue by ShicongWang** [GenjiB/ECLIPSE#6]

**Comment by ShicongWang** [comment]
04/10/2023 19:25:53 - INFO -   Weights from pretrained model not used in CLIP4Clip:

**Comment by GenjiB** [comment]
did you download weights from CLIP4Clip? Our model is initialized with a zero gate. So, the worst case is identical to CLIP4Clip

**Comment by ShicongWang** [comment]
I'd like to run the MSRVTT dataset using the ECLIPSE model, and I found that when evaluating the test set directly with the model, the R@1 value is normal at 26. However, when fine-tuning on the MSRVTT dataset, the R@1 value becomes very low, like 0.1 or 0.2, which is quite strange. Do you know the reason for this, or have you ever worked with the MSRVTT dataset? Could you please tell me the values you obtained when running the MSRVTT dataset?

**Comment by ShicongWang** [comment]
I am primarily interested in referencing and comparing your method with mine in a research paper, and I hope to receive your assistance in this matter

**Comment by GenjiB** [comment]
Based on my experience, the batchsize cannot be too small, and the learning rate cannot be too large. 

You can also check the learning rate at CLIP4Clip

**Comment by ShicongWang** [comment]
Yes, but after fine-tuning, the R@1 value is around 0.1, which is basically equivalent to random guessing. When not fine-tuned, the R@1 value is quite normal. It doesn't seem to be an issue with the parameter settings.

**Comment by ShicongWang** [comment]
Thank you, let me try your suggestion first.

**Comment by ShicongWang** [comment]
learning rate is 1e-4

**Comment by ShicongWang** [comment]
Thank you, you are correct.
